### DISCUSSION TITLE
image怎么显示不出来？

### DISCUSSION THREAD

**Discussion by jiapy97** [zeit-ui/react#622] [Q&A] [answered]
import { Button,Display,Image } from '@geist-ui/react'
function App() {
    return (
        <div className="App">
            这是App

            <Button>按钮</Button>
            <Display shadow caption="用于构建现代应用的开源设计系统。">
                <Image width="435px" height="200px" src="./images/LeetCode.png" />
            </Display>
        </div>
    );
}

![image](https://user-images.githubusercontent.com/57215916/132080248-0062f5df-fbaf-4320-8645-376aa1aa3966.png)

**Comment by harshcut** [comment, marked as answer]
It looks like the image does not exist in your `/public/images` directory. Check if you have an `404` error status in your browser console.

**Comment by jiapy97** [comment]
十分感谢，已经解决了，Thank you



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Harsh Karande ***@***.***&gt;
发送时间: 2021年9月4日 13:26
收件人: geist-org/react ***@***.***&gt;
抄送: Always--Positive ***@***.***&gt;, Author ***@***.***&gt;
主题: 回复：[geist-org/react] image怎么显示不出来？ (#622)
